reana_server/rest/users.py

```diff
@@ -148,98 +176,6 @@ def get_user():  # noqa
         return jsonify({"message": str(e)}), 500
 
 
-@blueprint.route("/users", methods=["POST"])
-def create_user():  # noqa
```

Comment:
Any specific reason why we get rid of this? Maybe we can specify it in the commit message.

Reply:
Mainly because we do not use it. I will specify in the commit message, thanks for pointing out :)